docs/release_notes/v0.8.0.md

```diff
@@ -0,0 +1,294 @@
+  
+# Dapr 0.8.0
+
+We're happy to announce the release of Dapr 0.8.0!
+
+**Highlights**
+
+* Added new components: MQTT, Apache Pulsar, AWS snsaqs pubsub
+* Support custom pubsub routes
+* Support Bi-directional bindings
+* Support Dapr API authentication
+* Support W3C trace header
+* Enhanced mDNS service discovery to support host name
+* Support default common components path for Dapr CLI
+* Added Python Actor SDK
+
+We would like to extend our thanks to all new and existing contributors who helped make this release happen.
+
+If you're new to Dapr, visit the [getting started](https://github.com/dapr/docs/tree/master/getting-started) page and familiarize yourself with Dapr.
+
+Docs have been updated with all the new features and changes of this release. To get started with new capabilities introduced in this release, go to the [Concepts](https://github.com/dapr/docs/tree/master/concepts) and the [How To section](https://github.com/dapr/docs/tree/master/howto#how-tos).
+
+> **Note: This release contains a few [breaking changes](#breaking-changes).**
+
+See [this](#upgrading-to-dapr-0.8.0) section on upgrading Dapr to version 0.8.0.
+
+## New in this release
+
+### Dapr Runtime
+
+**Added** custom routes for pub/sub (#breaking-changes)
+
+**Enriched** http and grpc status code conversion.
+
+**Added** Healthz server for control plane.
+
+**Refactored** gRPC service and protobuf message definitions. (#breaking-changes)
+
+**Fixed** Actor activation and method invocation logic to handle app restarts.
+
+**Fixed** Unpopulated metadata for input bindings for HTTP app channel.
+
+**Improved** client trace span generation calls.
+
+**Added** liveness config.
+
+**Added** disable Span Generations when tracing rate is set to 0.
+
+**Added** trace headers in API response, which allows users to correlate with
+
+**Enhanced** mDNS to support hostname and IPs. (Thanks @jjcollinge)
+
+**Added** mqtt pubsub provider to daprd. (Thanks @kumaya)
+
+**Added** Pulsar pubsub options. (Thanks @elyst)
+
+**Added** API Authentication.
+
+**Added** bi-directional bindings support. (#breaking-changes)
+
+**Added** Set Metadata option to state store Get API.
+
+**Added** opentelemetry sementic convention.
+
+**Added** snssqs pubsub provider.
+
+**Added** Context timeout to internal grpc dial.
+
+**Added** Linear interval retry for Internal gRPC client.
+
+**Added** pending actor locks metric.
+
+**Generated** low-cardinality metric method label.
+
+**Fixed** marshalling error for gRPC binary metadata.
+
+**Improved** component folder search behavior by looking at home directory. (#breaking-changes)
+
+**Fixed** log options for contrib logger and enable metrics only for middleware. (Thanks @ksashikumar)
+
+**Fixed** http method label for HTTP appchannel metric.
+
+
+### Components
+
+**Added** AWS SNS/SQS pubsub. (Thanks @khous)
+
+**Added** Apache Pulsar. (Thanks @elyst)
+
+**Added** MQTT pubsub. (Thanks @kumaya)
```

Comment:
Can we add links here for the new components: https://github.com/dapr/docs/blob/master/howto/setup-pub-sub-message-broker/setup-mqtt.md

Reply:
I will add the howto link in **Hightlight** section.

Reply:
@orizohar and @yaron2 added the link in hightlights